src/datagen/main/java/mekanism/client/lang/MekanismLangProvider.java

```diff
@@ -1282,6 +1285,7 @@ private void addMisc() {
         add(MekanismModules.EXCAVATION_ESCALATION_UNIT, "Excavation Escalation Unit", "Increases digging speed on any block.");
         add(MekanismModules.ATTACK_AMPLIFICATION_UNIT, "Attack Amplification Unit", "Amplifies melee attacks on players or mobs.");
         add(MekanismModules.SILK_TOUCH_UNIT, "Silk Touch Unit", "Allows all mined blocks to drop as themselves.");
+        add(MekanismModules.BLASTING_UNIT, "Blasting Unit", "Allows mining larger areas.");
```

Comment:
I need to think about this some more but I feel like we probably want a description that better matches some of the other descriptions.

Reply:
Maybe something like:
> Uses controlled explosions to destroy nearby blocks in the target plane.